libflagship/mqttapi.py

```diff
@@ -66,12 +71,13 @@ def on_message(self, client, userdata, msg, pkt, tail):
         pass
 
     @classmethod
-    def login(cls, printersn, username, password, key, ca_certs="ankermake-mqtt.crt", verify=True):
+    def login(cls, printersn, username, password, key, ca_certs=cert_path, verify=True):
         client = mqtt.Client()
 
         if verify:
             client.tls_set(ca_certs=ca_certs)
         else:
+            log.warning('Not verifying certificates is insecure')
```

Comment:
Could this be rephrased to be a positive statement instead of a double negative? 

By not verifying the certificate is insecure, is that the same as 'skipping certificate verification' ?

Reply:
I have made this more verbose and cleared up the language to provide suggested use case.